app/controllers/idv/phone_controller.rb

```diff
@@ -214,6 +214,21 @@ def async_state_done(async_state)
         **opt_in_analytics_properties,
       )
 
+      phone = idv_session.user_phone_confirmation_session&.phone ||
+              formatted_previous_phone_step_params_phone
```

Comment:
```suggestion
```
the `user_phone_confirmation_session` is the # submitted for the OTP which has not yet been attempted .... `formatted_previous_phone_step_params_phone` has been proofed by the vendor 👍🏿